templates/components/debug/debug_toolbar.html.twig

```diff
@@ -34,35 +34,35 @@
 {% set rand = rand|default(random()) %}
 
 {% macro widget_button(id, title, icon = null) %}
-   <li class="debug-toolbar-widget d-inline-block p-1" data-glpi-debug-widget-id="{{ id }}">
+   <li class="debug-toolbar-widget d-inline-block p-2" data-glpi-debug-widget-id="{{ id }}">
       <button type="button" class="btn btn-icon border-0 p-1" title="{{ title }}" data-bs-toggle="tab">
-         <span class="debug-text"></span>
          {% if icon != null %}
-            <i class="{{ icon }}"></i>
+            <i class="{{ icon }} me-1"></i>
          {% endif %}
+         <span class="debug-text"></span>
       </button>
    </li>
 {% endmacro %}
 
 <div id="debug-toolbar" class="position-sticky bottom-0 w-100 card" tabindex="0">
-   <div class="resize-handle my-n2" style="cursor: row-resize; height: 10px; z-index: 1030"></div>
-   <div class="d-flex flex-row">
-      <div class="debug-toolbar-badge d-flex py-2">
-         <button type="button" class="btn btn-icon border-0 p-0" onclick="GLPI.Debug.showDebugToolbar();">
+   <div class="resize-handle mt-n2" style="cursor: row-resize; height: 10px; z-index: 1030"></div>
+   <div class="d-flex flex-row align-items-center">
+      <div class="debug-toolbar-badge d-flex">
+         <button type="button" class="btn btn-icon border-0 px-3 opacity-100 debug-logo" onclick="GLPI.Debug.showDebugToolbar();" disabled>
             <i class="ti ti-bug"></i>
          </button>
       </div>
-      <div class="debug-toolbar-content p-1 d-flex w-100 justify-content-between">
+      <div class="debug-toolbar-content d-flex w-100 justify-content-between align-items-center">
          <ul class="debug-toolbar-widgets nav nav-tabs align-items-center border-0" data-bs-toggle="tabs">
-            {{ _self.widget_button('server_performance', _x('debug', 'Server performance')) }}
+            {{ _self.widget_button('server_performance', _x('debug', 'Server performance'), 'ti ti-clock-play') }}
             {{ _self.widget_button('sql', _x('debug', 'SQL Requests'), 'ti ti-database') }}
-            {{ _self.widget_button('requests', _x('debug', 'Requests'), 'ti ti-refresh') }}
+            {{ _self.widget_button('requests', _x('debug', 'HTTP Requests'), 'ti ti-refresh') }}
```

Comment:
Translations are frozen for 10.0.8.

Also, as discussed today in our channel, I wonder if debug bar strings should really be translated; it sounds like a waste of time for translators. Keeping debug bar in English only could make sense.

So, either we remove those translations, either you put the string back (and change it after 10.0.8 has been released).

Reply:
> I wonder if debug bar strings should really be translated

When we have to do support, we may sometimes ask what information is displayed in the debug bar. When I made translations on this part yesterday, I realized that some terms are difficult to translate and that it might be better to leave everything in English to avoid misunderstandings due to translation errors.

Reply:
Ok to remove translations for the debug bar if everyone agrees

Reply:
@orthagh @cconard96 do you think that translations could be removed here?

Reply:
I agree

Reply:
> @orthagh @cconard96 do you think that translations could be removed here?

Yes.

Reply:
Done in #15007 . You will have to rebase.